index.html

```diff
@@ -82,29 +82,22 @@
 <p>An open communication culture prevails. Although we develop and use Samply in a variety of independent research projects, we are committed to communicating ideas, problems and changes openly.</p>
-<p>For discussions, questions, complaints or feature requests, contact the developers via the <a href="mailto:dev@samply.de">Samply mailing list</a>, <a href="https://github.com/samply">Issue Tracker</a> or <a href="https://chat.germanbiobanknode.de">Samply chat</a>.</p>
+<p>For discussions, questions, complaints or feature requests, contact the developers via the <a href="mailto:dev@samply.de">Samply mailing list</a> or <a href="https://chat.germanbiobanknode.de">Samply chat</a>.</p>
 <p>Our working language for documentation is English. In meetings, we will choose our working language based on the preferences of the attendees.</p>
-<p>The Samply community is represented by the Samply committee. The members of the Samply committee decide by a majority about organisational topics like changes to the internal workflow. Modifications to the Samply manifest must be agreed unanimously by the committee members.</p>
-<p>The Samply committee consists of at least five, and at most nine members. If possible, at least two different active projects associated with the Samply software should be represented in the Samply committee.</p>
-<p>Current committee members may propose new committee members. Alternatively, community members may request admission to the committee. In both cases, the committee decides about the admission to the committee. Each committee member is elected for 12 months. Re-election is possible.</p>
-<p>Each Samply component is curated by one or more maintainers. Those who engage intensely in the development and enhancement of a component are predestined to become its maintainer (see <a href="https://github.com/samply/manifest/wiki/List-of-component-maintainers">list of component maintainers</a>).</p>
-<p>The Samply committee determines the component maintainers in agreement with potential candidates.</p>
-<p>Meetings should take place biannually and every member of the community is invited to participate. In addition, the Samply committee organises web conferences to inform the members about news, changes and the roadmap.</p>
-<p>The appointment and location of a meeting is organised by the committee. The meeting agenda will be prepared by all participating members.</p>
-<p>If you would like to join the Samply community, please register for our Samply mailing list by sending an email to <a href="mailto:join@samply.de">join(at)samply.de</a>. You can leave the community the same way. Communication via the mailing list is archived and might be made public.</p>
+<p>Each Samply component is curated by one or more maintainers. Those who engage intensely in the development and enhancement of a component are predestined to become its maintainer.</p>

```

Comment:
How is the maintainer decided? Can everyone apply? Who accepts the application?

Reply:
See latest push for updated suggestions.